metrics/prometheus_test.go

```diff
@@ -73,6 +73,29 @@ func TestPrometheusMetrics(t *testing.T) {
 			},
 			expCode: http.StatusOK,
 		},
+		{
+			name: "Filter creation latency",
+			addMetrics: func(pm *metrics.Prometheus) {
+				pm.MeasureFilterCreate("filter1", time.Now().Add(-15*time.Millisecond))
```

Comment:
Maybe add some second value, -time.Second, for example?

Reply:
It does not matter, we don't test that prometheus works correctly but rather that metric is registered.